docs/src/main/asciidoc/tls-registry-reference.adoc

```diff
@@ -757,22 +797,20 @@ spec:
     app.kubernetes.io/version: ...
   type: ClusterIP
 ----
-
++
 The annotation `service.beta.openshift.io/serving-cert-secret-name` instructs OpenShift to generate a certificate and store it in a secret named `my-tls-secret`.
 
-To use the stored certificate:
-
-. Add this annotation to your already running service:
+. To use the stored certificate, add this annotation to your already running service:
```

Comment:
I don't get this - the following snippet doesn't show how to use the certificate, it's just another way to have it generated (equivalent to the previous YAML-based snippet, but for applying to an already existing service)

Reply:
As for "Using the certificate", that's just the snippet described as

> Mount the secret as a volume in your pod by updating your _Deployment_ configuration:

Because by  using it, one usually  means mounting it into a deployment

Reply:
it's a prerequisite to use the certificate. This annotation actually trigger the generation of a certificate for this service. 

Maybe rephrase with:

```
To generate a certificate, add his annotation to your already created Kubernetes `service`
```

Reply:
But in the previous YAML snippet there's already 
```
service.beta.openshift.io/serving-cert-secret-name: my-tls-secret
```

so calling
```
oc annotate service hero-service \
     service.beta.openshift.io/serving-cert-secret-name=my-tls-secret
```
won't have any effect no? It's just a way to add it to an existing service rather than when creating it from scratch (YAML).

Reply:
let's me check, it may have been a mistake when I copy the content (I may have annotated it already)

Reply:
you are right, we need to remove the:
```
annotations:
    service.beta.openshift.io/serving-cert-secret-name: my-tls-secret
```

from the YAML of the service.

Reply:
Ok so @MichalMaler I would 

- change this line to
> To generate a certificate, add this annotation to your already created Kubernetes `service`

- remove the 
```
  annotations:
    service.beta.openshift.io/serving-cert-secret-name: my-tls-secret
``` 
from the previous YAML snippet

and move the sentence 

> The annotation `service.beta.openshift.io/serving-cert-secret-name` instructs OpenShift to generate a certificate and store it in a secret named `my-tls-secret`.

after the `oc annotate ...` snippet

Reply:
Done n done! Thank you both, @jmartisk @cescoffier